pandas/tests/groupby/test_groupby.py

```diff
@@ -2828,3 +2828,13 @@ def test_groupby_index_name_in_index_content(val_in, index, val_out):
     result = series.to_frame().groupby("blah").sum()
     expected = expected.to_frame()
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("n", [1, 10, 100, 1000])
```

Comment:
would it be worth putting in n=32 as this was the original value which triggered the bug?
https://github.com/pandas-dev/pandas/issues/50347

Reply:
For what it's worth it happened for n ≥ 32, but indeed it is relevant to add this case. Done.